test/extended/setup.sh

```diff
@@ -147,6 +150,9 @@ function os::test::extended::setup () {
 
 	os::log::info "Creating image streams"
 	oc create -n openshift -f "${OS_ROOT}/examples/image-streams/image-streams-centos7.json" --config="${ADMIN_KUBECONFIG}"
+
+	os::log::info "Creating sample app"
+	oc create -n openshift -f "${OS_ROOT}/examples/sample-app/application-template-stibuild.json" --config="${ADMIN_KUBECONFIG}"
```

Comment:
why's that?  it doesn't seem to assume anything the line directly above it doesn't also assume.